packages/mapviewer/src/modules/menu/components/advancedTools/ImportCatalogue/ProviderList.vue

```diff
@@ -57,26 +229,125 @@ defineExpose({ goToFirst })
             class="providers-list"
             data-cy="import-provider-list"
         >
+            <template v-for="node in treeData" :key="node.id">
+                <component
+                    :is="node.type === 'group' ? 'div' : 'div'"
+                    :class="node.type === 'group' ? 'providers-group' : 'providers-list-item'"
```

Comment:
could using something like
```
:class="`providers-${node.type}"
```
also work? that looks simpler to understand than a ternary operator in this context